template_content/smart_contracts/{{ contract_name }}/contract.algo.ts.jinja

```diff
@@ -2,10 +2,6 @@ import { Contract } from '@algorandfoundation/algorand-typescript'
 
 export class {{ contract_name.split('_')|map('capitalize')|join }} extends Contract {
   public hello(name: string): string {
-    return `${this.getHello()} ${name}`
-  }
-
-  private getHello() {
-    return 'Hello'
+    return `Hello, ${name}`
```

Comment:
It appears the template, even for the starter version, does not have semicolons, so that's what we want. Just want to test that this works once a project is `init` from the starter template. If not, it may require adding prettier config to the starter template.

Reply:
yeah - I formatted it according to the `prettier` config in the production template. I don't believe the starter template ships with eslint/prettier, pretty sure that's a setting passed in from the algokit CLI? If changes are needed here, let me know.

Reply:
you can test the output of this template if you run `algokit -v init --name playground --no-git --UNSAFE-SECURITY-accept-template-url --template-url . --template-url-ref HEAD --no-bootstrap` from the root of this repo btw. :)